docs/index.md

```diff
@@ -1,6 +1,7 @@
 ```{toctree}
 :hidden:
 :titlesonly:
+Getting Started <./getting-started/index>
```

Comment:
Probably better to have this under the "Examples" section rather than it's own heading. It could go under the "tutorials" bit.